redash/query_runner/cass.py

```diff
@@ -1,18 +1,19 @@
 import json
-import sys
 import logging
 
-from redash.query_runner import *
+from redash.query_runner import BaseQueryRunner, register
 from redash.utils import JSONEncoder
 
 logger = logging.getLogger(__name__)
 
 try:
-    from cassandra.cluster import Cluster
+    from cassandra.cluster import Cluster, Error
```

Comment:
not fully sure if this `Error` import works well. At least when I tested, it did not work and I have to remove it and replace the reference below by `ValueError`.